File: JavaScript/1010.js
// Problema: 1010 - Cálculo Simples  | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s] | Tempo: 0.024s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.trim().split('\n');

var total = 0;
lines.forEach(line => {
    let dados = line.split(' ');
    total += parseInt(dados[1]) * parseFloat(dados[2]);
});

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);

File: Java/1010.java
// Problema: 1010 - Cálculo Simples | Resposta: Accepted
// Linguagem: Java 19 [+2s]         | Tempo: 0.085s

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	
	static String[] pegaArray(BufferedReader in) throws IOException {
		return in.readLine().split(" ");
	}
	
	static float converteFloat(String valor) throws IOException {
		return Float.parseFloat(valor);
	}

    public static void main(String[] args) throws IOException {
    	InputStreamReader ir = new InputStreamReader(System.in);
	    BufferedReader in = new BufferedReader(ir);
	    
	    String[] linha;
	    float valor = 0;
	    
	    linha = pegaArray(in);
	    valor += converteFloat(linha[1]) * converteFloat(linha[2]);
	    linha = pegaArray(in);
	    valor += converteFloat(linha[1]) * converteFloat(linha[2]);

        System.out.printf("VALOR A PAGAR: R$ %.2f\n", valor);
        
        in.close();
        ir.close();
    }
}